SR2023/2022-09-29-confirm-team-details.md

```diff
@@ -0,0 +1,24 @@
+---
+to: SR2023 sign-ups
+subject: Confirming your details for SR2023
+---
+
+Hello!
+
+Thank you for your interest in Student Robotics 2023.
+
+Before we can confirm your place, we have a few questions and extra pieces of information we need.
```

Comment:
Maybe say something like "Thank you for your application, we would like to offer you a place. In order to do that.."
This would make it clear that they do essentially have a place, rather than this feeling like this is just another step and they may not have a place yet.